hbase/src/main/scala/geotrellis/spark/io/hbase/package.scala

```diff
@@ -20,8 +20,4 @@ import org.apache.hadoop.hbase.TableName
 import org.apache.hadoop.hbase.util.Bytes
 
 package object hbase {
```

Comment:
Can this file just be deleted?

Reply:
It should have been; I might have screwed something up during the rebase